tests/datastructures/test_headers.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from starlite.datastructures import CacheControlHeader
+from starlite.exceptions import ImproperlyConfiguredException
+
+
+def test_cache_control_to_header() -> None:
+    header = CacheControlHeader(max_age=10, private=True)
+    expected_header_values = ["max-age=10, private", "private, max-age=10"]
+    assert header.to_header() in expected_header_values
+    assert header.to_header(include_header_name=True) in [f"cache-control: {v}" for v in expected_header_values]
+
+
+def test_cache_control_from_header() -> None:
```

Comment:
Maybe test all possible fields here to make sure they're included / of the right type after parsing

Reply:
I'm not sure there's a reason to test all possible fields because there's no per-field logic. The logic is per field type - either "boolean" fields (like `public`, `no-store`, etc.) or "integer" fields (like `max-age`). This test tests both field types

Reply:
It would serve more to codify the field names, so changing their names / types would fail tests if not done in a backwards compatible way.

Reply:
Sure, fixed here: https://github.com/starlite-api/starlite/pull/601/commits/0f7dd0639e3401d984f94fc942036d191ce9aae7